apps/dashboard/components/chart/index.tsx

```diff
@@ -0,0 +1,153 @@
+"use client"
+import React, { useState } from "react"
+import { Card, Button, ToggleButtonGroup, Box } from "@mui/joy"
+import {
+  AreaChart,
+  Area,
+  XAxis,
+  YAxis,
+  Tooltip,
+  ResponsiveContainer,
+  TooltipProps,
+} from "recharts"
+
+import CircularProgress from "@mui/joy/CircularProgress"
+
+import useProcessedTransactionsForChart from "@/hook/use-processed-transactions-for-chart"
+import { TransactionEdge } from "@/services/graphql/generated"
+
+const CustomTooltip = ({
+  active,
+  payload,
+  walletCurrency,
+}: TooltipProps<number, string> & { walletCurrency: "USD" | "Sats" }) => {
```

Comment:
There is a lack of consistency in the capitalization. It should probably either be `USD | BTC` or `CENTS | SATS`

```suggestion
}: TooltipProps<number, string> & { walletCurrency: "USD" | "Sats" }) => {
```